### PR TITLE
CHE-1104: Fix activation  breakpoints in the Java Debugger

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
**_1 Upvote**_ Signed-off-by: Aleksandr Andrienko aandrienko@codenvy.com

### CODE REVIEW

**Comment by vparfonov** [comment]
+1

File: plugins/plugin-java-debugger/che-plugin-java-debugger-ide/src/main/java/org/eclipse/che/plugin/jdb/ide/fqn/JavaFqnResolver.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public String resolveFqn(@NotNull final VirtualFile file) {
         String projectBuilder = getProjectBuilder(projectNode);
         Map<String, List<String>> attributes = projectNode.getProjectConfig().getAttributes();
 
-        sourceFolders.addAll(attributes.containsKey(projectBuilder + ".source.folder") ? attributes.get(projectBuilder + ".source.folder")
+        sourceFolders.addAll(attributes.containsKey("java.source.folder") ? attributes.get("java.source.folder")
```
Comment (azatsarynnyy) [inline plugins/plugin-java-debugger/che-plugin-java-debugger-ide/src/main/java/org/eclipse/che/plugin/jdb/ide/fqn/JavaFqnResolver.java]: you should use special constant which contains this attribute's name
`org.eclipse.che.ide.ext.java.shared.Constants#SOURCE_FOLDER`

File: plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/projecttree/JavaSourceFolderUtil.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public static boolean isSourceFolder(ItemReference item, HasProjectConfig projec
                 final String projectPath = projectConfig.getPath();
                 final String itemPath = item.getPath();
 
-                List<String> sourceFolders = attributes.get(projectBuilder + ".source.folder");
+                List<String> sourceFolders = attributes.get("java.source.folder");
```
Comment (azatsarynnyy) [inline plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/projecttree/JavaSourceFolderUtil.java]: see https://github.com/eclipse/che/pull/1298#discussion_r63864984

File: plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/projecttree/JavaSourceFolderUtil.java [hunk 3]
```diff
@@ -104,7 +104,7 @@ private static boolean isSourceFolder(@Null List<String> sourceFolders, String p
         String projectPath = removeEndingPathSeparator(projectConfig.getPath());
         Map<String, List<String>> attributes = projectConfig.getAttributes();
 
-        List<String> sourceFolders = attributes.get(projectBuilder + ".source.folder");
+        List<String> sourceFolders = attributes.get("java.source.folder");
```
Comment (azatsarynnyy) [inline plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/projecttree/JavaSourceFolderUtil.java]: see https://github.com/eclipse/che/pull/1298#discussion_r63864984

File: plugins/plugin-java/che-plugin-java-ext-lang-client/src/test/java/org/eclipse/che/ide/ext/java/client/projecttree/JavaSourceFolderUtilTest.java [hunk 4]
```diff
@@ -34,7 +34,7 @@ public void setUp() throws Exception {
         projectConfig.setPath("/test");
         projectConfig.setType("maven");
         projectConfig.setAttributes(new HashMap<String, List<String>>() {{
-            put("maven.source.folder", Collections.singletonList("src/main/java"));
+            put("java.source.folder", Collections.singletonList("src/main/java"));
```
Comment (azatsarynnyy) [inline plugins/plugin-java/che-plugin-java-ext-lang-client/src/test/java/org/eclipse/che/ide/ext/java/client/projecttree/JavaSourceFolderUtilTest.java]: see https://github.com/eclipse/che/pull/1298#discussion_r63864984

Comment (AndrienkoAleksandr) [inline plugins/plugin-java/che-plugin-java-ext-lang-client/src/test/java/org/eclipse/che/ide/ext/java/client/projecttree/JavaSourceFolderUtilTest.java]: Ok, You are right, thank you

**Pull request merged**